<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaceLoginAndRegister</class>
 <widget class="QDialog" name="FaceLoginAndRegister">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1251</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/new/icon/D:/image/bg.png);</string>
  </property>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>291</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Face Anti-spoofing</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelRegister">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>0</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="photoShow">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>521</width>
     <height>441</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>360</y>
     <width>371</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_16">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>   Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelName_3">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_17">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>   Face confidence:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelFaCo_3">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_18">
      <item>
       <widget class="QLabel" name="label_14">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>   Liveface:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelLive_3">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_19">
      <item>
       <widget class="QLabel" name="label_15">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>   Live confidence:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelLiCo_3">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_20">
      <item>
       <widget class="QLabel" name="label_16">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>   Detection time:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelDeTi_3">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="labelLogin">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>0</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNewName">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>210</y>
     <width>371</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonHome">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>10</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelRegisterWord_3">
   <property name="geometry">
    <rect>
     <x>506</x>
     <y>100</y>
     <width>169</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Face Register</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLoginWord_3">
   <property name="geometry">
    <rect>
     <x>301</x>
     <y>100</y>
     <width>143</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Face Detect</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonRegister_3">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>130</y>
     <width>23</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonLogin_3">
   <property name="geometry">
    <rect>
     <x>331</x>
     <y>134</y>
     <width>23</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
